### PR TITLE
SDOR-50: Draft config for ELB and ASG

### PR DESCRIPTION
**Pull request by babinskiy**

### CODE REVIEW

**Comment by artemuskov** [comment]
What about use nginx instead of Apache httpd?

**Comment by babinskiy** [comment]
so far as we dont have application, there is very simple draft userdata.
Feel free to replace it to anything you want

File: ops/app_ELB.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# Parameters
+################################################################################
+Parameters:
+  Environment:
+    Type: String
+    Description: Environment (qa, prod).
+    AllowedValues: [qa, prod]
+    ConstraintDescription: Must be a valid environment name
+
+  Owner:
```
Comment (artemuskov) [inline ops/app_ELB.yml]: The same as in previous template

File: ops/deploy_app.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# Simple script for deploing all with nessessary paramerers
+
+aws s3 cp ./app.pp s3://30daysdevops/nick/
+
+# Create ELB Stack
+aws --output text cloudformation create-stack \
```
Comment (artemuskov) [inline ops/deploy_app.sh]: Please add tags: Key=service, Value=load-balancer

File: ops/deploy_app.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    --template-body file://app_ELB.yml \
+    --stack-name nick-app-ELB \
+    --parameters ParameterKey=Environment,ParameterValue=qa
+
+# Create connected ASG stack
+aws --output text cloudformation create-stack \
```
Comment (artemuskov) [inline ops/deploy_app.sh]: Please add tags: Key=service, Value=auto-scaling-group

**Comment by artemuskov** [comment]
As jira have special plugins to monitoring pool requests, it should be named one ticket - one branch - one PR

File: ops/deploy_app.sh (hunk 3 above)
Comment (babinskiy) [reply ops/deploy_app.sh]: do you mean, tags on CF Stack? or on exactly Auto-Scaling Group?

Comment (artemuskov) [reply ops/deploy_app.sh]: on CF stack

Comment (babinskiy) [reply ops/deploy_app.sh]: OK, updated

**Pull request merged**